src/Elems/itemList.js

```diff
@@ -0,0 +1,210 @@
+                        saveTodos={this.props.saveTodos}
+                        doneTodos={this.props.doneTodos}/>
+                     );    
+            });
+            return undoneData;
+        }
+        else if (filterState === 'done') {
+            const doneArray = this.props.data.filter(el => el.status === true)
+            const doneData  = doneArray.map((item) => {
+                return (
+                    <Item 
+                        key={item.id}
+                        id={item.id} 
+                        data={item}
+                        delTodos={this.delTodos}
+                        saveTodos={this.props.saveTodos}
+                        doneTodos={this.props.doneTodos}/>
+                )    
+            });
+            return doneData;
+        };
+    };
+        return ( 
+            <div 
+            className='list list-group'>
+                    <div
+                        className='allDoneUndone'>
+                        <button
+                            className='allDoneBtn'
+                            onClick={this.onMarkAllBtnClick}>
+                            ⌄
+                        </button>
+                    </div>                 
+                    {changeData()}                    
+                    <div
+                        className='buttons list-group-item'>
+                            <p 
+                                className='footerP'>
+                                All items: {this.props.data.length}</p>
+                            <button
+                                className="AUDBtn"
+                                id={this.state.allChkd}
+                                onClick={this.onAllListBtnClick}>
+                                All
+                            </button>
+                            <button
+                                className="AUDBtn"
+                                id={this.state.undoneChkd}
+                                onClick={this.onUndoneListBtnClick}>
+                                Active 
+                            </button>
+                            <button
+                                className="AUDBtn"
+                                id={this.state.doneChkd}
+                                onClick={this.onDoneListBtnClick}>
+                                Completed 
+                            </button>
+                            <button
+                                className="AUDBtn"
+                                id='clearCompleted'
+                                onClick={this.onDeleteAllDoneBtnClick}>
+                                Clear completed
+                            </button>                       
+                    </div>
+                    <div
+                        className='footer list-group-item'>                    
+                    </div>  
+            </div>
+        );
+    };
+};
```

Comment:
Use soft tabs (space character) set to 2 spaces.
Follow alignment styles for JSX syntax